### PR TITLE
Added the support for PyQt4 GUI 

### PR DESCRIPTION
**Pull request by yashrsharma44**
Added the support for GUI using PyQt4.
Support for #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,24 @@ Other than requiring Python3.x , you require the following libraries:<br>
...
    
+# For the GUI Version
+
+For the GUI Version, follow the steps
+1. Follow the preceeding steps uptil 2.
+2. Use a virtual environment `virtualenv --python=python3.5 --no-site-packages venv` 
+3. PyQt4 is not supported in Virtual Environment, so you need to setup PyQt manually in the `virtualenv`.
+4. Two dependencies need to be downloaded,PyQt4 and SIP.
+5. Download the binaries from the given website - 
+    PYQT4 - https://www.riverbankcomputing.com/software/pyqt/download  (Choose the Source Packages)
```
Comment (Parth-Vader) [inline README.md]: Add both of them in MarkDown Link form. [like this](github.com)

File: README.md [hunk 2]
```diff
@@ -59,6 +59,24 @@ Other than requiring Python3.x , you require the following libraries:<br>
...
+    SIP - https://www.riverbankcomputing.com/software/sip/download
+    
+6. In the folder, make a folder named build, and extract the PyQt4 and SIP, in the build folder.
+7. You can refer this youtube tutorial for rest part - https://www.youtube.com/watch?v=6NDVP5inphM
```
Comment (Parth-Vader) [inline README.md]: This is not how you write a README. You need to give full instructions - you can't just link a youtube video tutorial. Although you can write towards the end that the source was this video.

**Comment by Parth-Vader** [comment]
Impressive work! But it's not working in my system.
I'm getting this error : `AttributeError: 'QString' object has no attribute 'startswith'`.

**Comment by yashrsharma44** [comment]
Can you state if you are using `virtualenv` or not?
Also try using python3.
IMHO, try to crosscheck with the steps that I have stated in the README.md

**Comment by Parth-Vader** [comment]
@yashrsharma44 My bad, I accidentally used an old virtualenv with Python 2.7.
Great work :smile: 
Should this be added to the `master` branch or another branch `gui`? If to the master, I would like both the `graph.py` file to exist and this too. Also I'm requesting some changes.

File: main.py [hunk 3]
```diff
@@ -0,0 +1,142 @@
+from PyQt4 import QtGui
```
Comment (Parth-Vader) [inline main.py:1]: After this, another commit was included (for Windows functionality) - [this.](https://github.com/Parth-Vader/FB-Spider/commit/d9a373712be1a0f492c53a153c59e9ac83b886d8)
Make the suitable changes here too.

File: README.md [hunk 4]
```diff
@@ -59,6 +59,24 @@ Other than requiring Python3.x , you require the following libraries:<br>
    A file called 'Table.html' would be made in the folder itself.
    You can open it and see the table.
```
Comment (Parth-Vader) [inline README.md:63]: If you want these to be merged in `master`, you also need to add the latest `graph.py` file.
Also, changes in the README.md should be made clarifying these 2 paths to run the program,

**Review by Parth-Vader — CHANGES_REQUESTED** [review]
Added some minor comments about adding `graph.py`, making suitable changes in README and adding some lines in `main.py`.

**Comment by yashrsharma44** [comment]
Thanks for your views. I will start working in it, in order to make sure it is merged in the master branch.
Sorry for not completing the README.md. I will make sure to complete them.

**Comment by yashrsharma44** [comment]
Added the changes.

**Review by Parth-Vader — APPROVED** [review]

**Comment by Parth-Vader** [comment]
This was surely a great PR @yashrsharma44!
I would also want you to add the GIFs for GUI version, and maybe you could also come up with an icon for FB-Spider :wink:

**Pull request merged**